modules/ROOT/pages/amazon/amazon-sns-connector-examples.adoc

```diff
@@ -0,0 +1,274 @@
+
+. In the Mule Palette, search for *Logger*, and drag next to the Amazon SNS connector (*Publish*).
+. Configure the *Logger* component as follows:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|*Display Name* |`Logger` (or any other name you prefer)
+|*Message* |`#[payload]`
+|*Level* |`INFO`
+|===
++
+image::amazon/amazon-sns-logger.png[sns-logger]
+
+[[receive-sns]]
+=== Receive SNS Messages
+
+In the same example, add a second flow that enables Amazon SQS to receive messages published by Amazon SNS:
+
+. In the Mule Palette, search for *Flow* and drag the *Flow* component to the canvas.
+. In the Mule Palette, search for *SQS*, select the *Receivemessages* operation, and drag it to the left of the *Flow* component.
+. Select the new component.
+. Click *+* next to the *Connector configuration* field to add a new Amazon SQS connector global element.
+. Configure the global element as follows:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Description|Value
+|Name |Enter a name for the configuration to reference it. |<Configuration_Name>
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the account. |Config_Access_Key
+|Secret Key |Key that plays the role of a password. |<Config_Secret_Key>
+|Default Global Queue URL |Set the queue URL |<Config_Queue_URL>.
+|Region Endpoint |Queue region
+|===
++
+For example:
++
+image::amazon/amazon-sns-sqs-config.png[sns-sqs-config]
++
+Select the *Configuration XML* tab to view the corresponding XML:
++
+[source,xml]
+----
+<sqs:config name="Amazon_SQS_Configuration"
+  doc:name="Amazon SQS Configuration"
+  defaultQueueUrl="https://sqs.us-east-1.amazonaws.com/sns-test" >
+<sqs:basic-connection accessKey="AAAJSIIXXJLRPC632UV"
+  secretKey="ZZZo7kc4luuvsCjDnUjftK" />
+</sqs:config>
+----
++
+Make sure the SQS queue in the configuration is subscribed to the SNS topic.
++
+. Click *Test Connection* to confirm that Mule can connect with the SQS instance:
+** If the connection is successful, click *OK* to save it.
+** If the connection is not successful, review or correct any incorrect parameters, and then test again.
+. In the properties editor of the Amazon SQS connector, configure the remaining parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+2+|Basic Settings
+|Display Name |`Amazon SQS (Streaming)`, or any other name you prefer.
+|Connector configuration*|`Amazon_SQS_Configuration` (the reference name to the global element you have created).
+|Other fields in the *General* group | Default values
+|===
++
+Select the *Configuration XML* tab to view the corresponding XML:
++
+[source,xml]
+----
+<sqs:receivemessages doc:name="Amazon SQS (Streaming)"
+  doc:id="d3b24d1d-6976-4c8b-bd99-3ab84dd1811d"
+  config-ref="Amazon_SQS_Configuration"/>
+----
++
+. Add a *Logger* scope after the Amazon SQS connector to print the data passed by the *Receive* operation in the Mule Console. Configure the *Logger* as follows:
++
+[%header%autowidth.spread]
```

Comment:
If you used `[%header,cols="30s,70a"]` the "s" operator (next to 30) would automatically bold all values in the column.